src/browser.coffee

```diff
@@ -18,6 +17,10 @@ CoffeeScript.run = (code, options = {}) ->
   options.shiftLine = on
   Function(compile code, options)()
 
+CoffeeScript.register = ->
```

Comment:
On a separate note, I also think `runScripts` method in the `browser.coffee` could be used as browser's `register` implementation. It's logically sound. But it will only work if the browser tests are actually running in browser. Right now it will only break the test when trying to call it in node environment.